exasol/analytics/udf/communication/peer_communicator/background_listener_thread.py

```diff
@@ -96,6 +98,50 @@ def create_background_peer_state_builder() -> BackgroundPeerStateBuilder:
     return background_peer_state_factory
 
 
+@dataclasses.dataclass
+class RuntimeSockets:
+    in_control: Socket
+    out_control: Socket
+    listen: Socket
+    listen_port: int
+    poller: Poller
+
+    @classmethod
+    def create(
+        cls,
```

Comment:
Maybe define a generics for `cls`, see https://docs.python.org/3/library/typing.html#generics

Reply:
This is a class method, compare to `self`.